README.md

```diff
@@ -1110,6 +1110,12 @@ messagingSession.addObserver(observer);
 messagingSession.start();
 ```
 
+## WebRTC Resources
```

Comment:
Should we put this on the top before use cases? Not sure if it is really discoverable here.

Reply:
Yes, agree moved it top before installation step.